az-digital/az_quickstart
# Closes #2160 Fix ordering of regions on Block Layout page

## Description
Updates the list of regions in az_barrio.info.yml so that the Block Layout page will display the regions in the correct order.

Changes made to az_barrio.info.yml:
- Moved "Sidebar first" and "Sidebar second" directly above "Content"
- Switched the order of "Breadcrumb" and "Content Top"
- Moved "Full width content bottom" below "Content"
- Moved "Content bottom" below "Content 3-column stacked(3)"

## Related issues




#2160 

## How to test


Visit the Block Layout page at /admin/structure/block and confirm that the regions match the block region demonstration at admin/structure/block/demo/az_barrio.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
